client/src/pages/CreateEnergiser.js

```diff
@@ -0,0 +1,90 @@
+import { useState } from "react";
+import Navbar from "./components/Navbar";
+import Footer from "./components/Footer";
+import "./CreateEnergiser.css";
+
+
+const CreateEnergiser = () =>{
+// const [addingEnergiser, setAddingEnergiser] = useState(false);
+const [energiserName, setEnergiserName] = useState("");
+const [enrgiserDescription, setEnergiserDescription] = useState("");
+
+const handleEnergiserAdder = (e) => {
+    e.preventDefault();
+
+    let energizer = {
+      name: energiserName,
+      description: enrgiserDescription,
+    };
+
+     fetch("/api/energizers",{
+            method:"POST",
+            headers:{ "Content-Type": "application/json" },
+            body: JSON.stringify(energizer),
+          })
+          .then((res) => res.json())
+          .then((data) => {
+            console.log(data);
+          }).
+          catch((error)=> {
+        console.warn(error);
```

Comment:
Should the user not be advised of any errors (through something visible in the UI)?